components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -430,23 +478,33 @@ CommandLineArguments::parse_arguments(int argc, char const** argv) {
                 std::cerr << std::endl;
 
                 std::cerr << "  # Search archives in archives-dir for logs matching a KQL query"
-                             R"( "level: INFO" and output to MongoDB)"
+                             R"( "level: INFO" and output to the results cache)"
                           << std::endl;
-                std::cerr << "  " << m_program_name
-                          << " s --mongodb-uri mongodb://127.0.0.1:27017/test --mongodb-collection"
-                             R"( test archives-dir "level: INFO")"
+                std::cerr << "  " << m_program_name << R"( s archives-dir "level: INFO")"
+                          << " " << cResultsCacheOutputHandlerName
+                          << " --uri mongodb://127.0.0.1:27017/test"
+                             " --collection test"
                           << std::endl;
 
```

Comment:
Should we add some examples of the reducer usage (and network output usage) in this PR?